plugins/plugin-react-refresh/test/__snapshots__/plugin.test.js.snap

```diff
@@ -370,7 +370,7 @@ function register(type, id) {
     allFamiliesByType.set(type, family); // Visit inner types because we might not have registered them.
 
     if (typeof type === 'object' && type !== null) {
-      switch (type.$typeof) {
+      switch (type.$$typeof) {
```

Comment:
why this change? is this expected? is this the bug being fixed?

Reply:
yea. It's expected. 'Cause react-refresh source code needs to check `$$typeof`, but the previous version of `@snowpack/plugin-react-refresh` change `$$` to `$`, due to `$&` in replacer function.
see: [https://github.com/facebook/react/blob/master/packages/react-refresh/src/ReactFreshRuntime.js](https://github.com/facebook/react/blob/master/packages/react-refresh/src/ReactFreshRuntime.js)